scripts/benchmark/benchmark-pull-request.sh

```diff
@@ -11,11 +11,51 @@
 # This script is used in OpenSearch Core repo github actions
 # To trigger Jenkins Gradle Check from a PR
 
-
+# Default values
 JENKINS_URL="https://build.ci.opensearch.org"
-TRIGGER_TOKEN=$1
+TRIGGER_TOKEN=""
+GITHUB_USER=""
+GITHUB_TOKEN=""
+
+while getopts "u:t:p:" opt; do
```

Comment:
would this affect the way script is executed? Do we need to pass some flags as well now?

Reply:
Yes just seems cleaner instead of positional args. Will be making change in core repo before merging this.
It will look like this
```bash
./benchmark-pull-request.sh -t genericToken -u user -p githubToken